velox/exec/tests/AggregationFuzzer.cpp

```diff
@@ -491,6 +500,31 @@ std::vector<RowVectorPtr> AggregationFuzzer::generateInputData(
   return input;
 }
 
+std::vector<RowVectorPtr> AggregationFuzzer::generateInputDataWithId(
+    std::vector<std::string> names,
+    std::vector<TypePtr> types) {
+  names.push_back("id");
+  types.push_back(BIGINT());
+  auto inputType = ROW(std::move(names), std::move(types));
+
+  std::vector<RowVectorPtr> input;
+  auto size = vectorFuzzer_.getOptions().vectorSize;
+  int64_t identifier = 0;
+  for (auto i = 0; i < FLAGS_num_batches; ++i) {
+    std::vector<VectorPtr> children;
+    for (auto i = 0; i + 1 < inputType->size(); ++i) {
+      children.push_back(vectorFuzzer_.fuzz(inputType->childAt(i), size));
+    }
+    velox::test::VectorMaker vectorMaker{pool_.get()};
```

Comment:
Do we need a separate instance for VectorMaker for each batch? Perhaps, use a single instance defined before the loop.